test/Vault.spec.ts

```diff
@@ -320,6 +320,12 @@ describe('Vault', () => {
       expect(await vault.treasury()).to.be.equal(TREASURY);
       expect(await vault.perfFeePct()).to.be.equal(PERFORMANCE_FEE_PCT);
     });
+
+    it('check the emission of the TreasuryUpdate event', async () => {
```

Comment:
Pedantic but I would capitalize the `c` at the start of this test case description

Reply:
I would suggest changing this to `emits the TreasuryUpdated event`

Reply:
@joaop21 fair point i just saw the test above and that one is capitalized nvm me :+1:

Reply:
guys, tests should NOT start with a capitalize letter, the whole idea is to make it a readable line "it emits a TreasuryUpdated event", or "it checks the emission of the TreasuryUpdate event". But yeah, the first one is better